index.html

```diff
@@ -468,13 +488,29 @@ <h3 id="liaisons"> Liaisons </h3>
         <h3 id="external-groups"> External Groups </h3>
         <p> While the Presentation API does not have a direct dependency on any
           given set of protocols, the following is a tentative list of external
-          bodies the Working Group should collaborate with to allow the
-          Presentation API to be implemented on top of widely deployed
-          attachment methods for connected displays: </p>
+          bodies the Working Group should collaborate with to develop the Open
+          Screen Protocol and allow the Presentation API and Remote Playback API
+          to be implemented on top of widely deployed attachment methods for
+        connected displays: </p>
         <dl>
           <dt><a href="http://www.ietf.org" id="ietf">IETF</a></dt>
-          <dd> The IETF develops home network protocols that presentation displays
-            may support. </dd>
+          <dd> IETF develops home network protocols that presentation displays
+            may support and at the core of the Open Screen Protocol. The Second
```

Comment:
This is unclear, needs some rewording. Maybe remove "and"?

Reply:
Indeed, the sentence is confusing. I think what I was trying to capture is that the IETF develops more home network protocols that agents could support than the ones the Open Screen Protocol makes use of. I'll push an update.

Reply:
Thanks, Fix LGTM.